index.html

```diff
@@ -16,7 +16,7 @@
 
 <body id="index-body">
     <div id="header-container">
-        <img class="logo" src="images/3D-IMAGE-LOGO.jpg" alt="3D Logo">
+     <img class="logo" src="images/3D-IMAGE-LOGO.jpg" alt="3D Logo">
         <h2 class="services"> SERVICES</h2>
```

Comment:
### Add an <a> tag.

- These h2 classes are suppose to be links, so please add an <a> tag.